tekton/resources/checks/check-pr-has-kind-label.yaml

```diff
@@ -0,0 +1,216 @@
+    kind: TaskRun
+    metadata:
+      name: check-pr-labels-$(uid)
+    spec:
+      taskSpec:
+        inputs:
+          resources:
+          - name: source
+            type: git
+          - name: pr
+            type: pullRequest
+          params:
+          - name: labels 
+            description: The labels attached to the Pull Request
+          - name: gitRepository
+            description: The URL of the git repository
+          - name: pullRequestUrl
+            description: The URL of the Pull Request
+        outputs:
+          resources:
+          - name: pr
+            type: pullRequest
+        volumes:
+        - name: label-config-v2
+          configMap:
+            name: label-config-v2
+        steps:
+        - name: copy-pr-to-output
+          image: busybox
+          script: |
+            #!/bin/sh
+            mkdir -p $(outputs.resources.pr.path)
+            cp -r $(inputs.resources.pr.path)/* $(outputs.resources.pr.path)/
+        - name: install-pyyaml
+          image: python:3-alpine
+          script: |
+            pip install pyyaml --user
+        - name: check-labels
+          image: python:3-alpine
+          volumeMounts:
+            - name: label-config-v2
+              mountPath: /etc/config
+          script: |
+            #!/usr/bin/env python
+            
+            import json
+            import yaml
+
+            prLabels = json.loads('$(params.labels)')
+            labelNames = list(map(lambda e: e["name"], prLabels))
+            kindLabels = list(filter(lambda e: "kind" in e, labelNames))
+
+            availableLabels = None
+            with open("/etc/config/labels.yaml", "r") as stream:
+              availableLabels = yaml.safe_load(stream)["default"]["labels"]
+
+            availableKindLabels = list(filter(lambda e: "kind/" in e["name"], availableLabels))
+            availableKindNamesAndDescriptions = map(lambda e: "`" +str(e["name"])+ "`" + ":  " + str(e["description"]), availableKindLabels)
+
+            comment_template=""
+            if (len(kindLabels) > 1 or len(kindLabels) == 0):
+              comment_template += """
+            **This PR cannot be merged:** expecting exactly one kind/ label
+
+            <details>
+
+            Available `kind/` labels are:
+
+            """
+            
+              for i in availableKindNamesAndDescriptions:
+                comment_template += i + "\n"
+
+              comment_template += """
+
+            </details>
+            """
+              new_comment_path = "$(outputs.resources.pr.path)/comments/new.json"
+              comment_body = dict(body=comment_template)
+              with open(new_comment_path, "w") as comment:
+                json.dump(comment_body, comment)
+
+              fail_check = {
+                "State": "failure",
+                "Label": "check-pr-has-kind-label",
+                "Desc": "The PR requires exactly one kind/ label to merge"
```

Comment:
NIT: We might want to add a text with a sample command to add the label, e.g. "/kind [feature|documentation|...]"
but that's something we can also add on top

Reply:
When the check fails, the issue comment includes a list of possible `kind/` labels to add.

Reply:
oh, nice